pyanaconda/modules/storage/partitioning/interactive/utils.py

```diff
@@ -711,6 +705,31 @@ def get_device_raid_level_name(device):
     return raid_level.name if raid_level else ""
 
 
+def get_container_raid_level(container):
+    """Get the RAID level of the given container.
+
+    :param container: a container
+    :return: a RAID level
+    """
+    raid_level = get_device_raid_level(container)
+
+    if raid_level:
+        return raid_level
+
+    device = container.raw_device
+
+    if hasattr(device, "lvs") and len(device.parents) == 1:
```

Comment:
Could you please make this condition more self-explanatory?

Reply:
This is copy-pasted from `get_device_raid_level`. @vojtechtrefny, is there a simpler way to implement `get_container_raid_level`?

Reply:
I didn't meant to change the whole algorithm. I was just thinking about something like:
```
is_lvm_container = hasattr(device, "lvs") and len(device.parents) == 1

if is_lvm_container:....
```

The point is that it's not easy to know what you are trying to find out by looking on number of parents and if there is "lvs" attribute.

Reply:
I have added comments.

Reply:
What if it's a btrfs subvolume?

Reply:
That should be covered by `get_device_raid_level`.